dist/tools/backport_pr/backport_pr.py

```diff
@@ -135,6 +135,13 @@ def main():
     if status != 200:
         print("Could not retrieve user: {}".format(user['message']))
         exit(1)
+    scopes = g.getheaders()
+    scopes = dict(scopes)["X-OAuth-Scopes"]
+    scopes = scopes.split(',')
+    scopes = [x.strip() for x in scopes]
```

Comment:
This could be simplified (but maybe harder to read) with

```py
    scopes = re.split(r"\s+,\s+", scopes)
```

Reply:
I will leave the split as it is  for now since there is no `import re` yet , and I like easier to read.